WordPress/Classes/ViewRelated/Activity/ActivityListViewController.swift

```diff
@@ -178,6 +175,12 @@ extension ActivityListViewController {
     }
 
     fileprivate func checkStatusDelayedForRestoreID(_ restoreID: String) {
+        delayedRetryAttempt = delayedRetryAttempt + 1
+        guard delayedRetryAttempt < Configuration.maxRetries else {
+            delay.reset()
+            showErrorFetchingRestoreStatus()
```

Comment:
Should this also set `tableView.isUserInteractionEnabled = true`?

Reply:
Yup, added and merging. Thanks for the review :)